src/app/[locale]/(app)/_components/SharePopup.tsx

```diff
@@ -56,6 +56,7 @@ function useImageData() {
         count: formatNumber(guild.message_count),
       };
     }),
+    t: useTranslation().t,
```

Comment:
I'd prefer to have a `useTranslation` in `useGenerateImg` and pass `t` to `StaticShareImage` directly, not through `generate`

Reply:
As for this one, yes I'll fix that !👍

Reply:
@florian-lefebvre After reviewing this i didn't find any other way to do this, or maybe i did not understand your request 🤔

Reply:
You know what, that's not a big deal let's keep it like this

Reply:
Alright !